src/index.js

```diff
@@ -26,6 +26,9 @@ export function startStimulusApp(context) {
         }
 
         symfonyControllers[controllerName].then((module) => {
+            // Normalize the controller name: remove the initial @ and use Stimulus format
+            controllerName = controllerName.substr(1).replace(/_/g, "-").replace(/\//g, "--");
```

Comment:
Do the controllerNames always start with an `@` or could there be another char?

```suggestion
controllerName = controllerName.replace(/^@/, "-").replace(/_/g, "-").replace(/\//g, "--");
```

Reply:
It always starts with `@` as it's generated using Flex